tutorials/notebooks/pytorch/ptq/pytorch_mobilevit_xs_for_imx500.ipynb

```diff
@@ -0,0 +1,384 @@
+    "\n",
+    "\n",
+    "## Summary\n",
+    "\n",
+    "In this tutorial we will cover:\n",
+    "\n",
+    "1. Post-Training Quantization using MCT of PyTorch classification model.\n",
+    "2. Data preparation - loading and preprocessing validation and representative datasets from ImageNet.\n",
+    "3. Accuracy evaluation of the floating-point and the quantized models."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d74f9c855ec54081",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "## Setup\n",
+    "### Install the relevant packages"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7c7fa04c9903736f",
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "!pip install -q torch\n",
+    "!pip install onnx\n",
+    "!pip install timm\n",
+    "!pip install 'huggingface-hub>=0.21.0'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "57717bc8f59a0d85",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    " Clone a copy of the [MCT](https://github.com/sony/model_optimization) (Model Compression Toolkit) into your current directory. This step ensures that you have access to [MCT Models Garden](https://github.com/sony/model_optimization/tree/main/tutorials/mct_model_garden) folder which contains all the necessary utility functions for this tutorial.\n",
+    "  **It's important to note that we use the most up-to-date MCT code available.**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9728247bc20d0600",
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "!git clone https://github.com/sony/model_optimization.git local_mct\n",
+    "!pip install -r ./local_mct/requirements.txt\n",
+    "import sys\n",
+    "sys.path.insert(0,\"./local_mct\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7a1038b9fd98bba2",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "### Download ImageNet validation set"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8bea492d71b4060f",
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "!mkdir imagenet\n",
+    "!mkdir imagenet/val\n",
+    "!wget https://image-net.org/data/ILSVRC/2012/ILSVRC2012_img_val.tar\n",
+    "!mv ILSVRC2012_img_val.tar imagenet/val\n",
+    "!tar -xvf imagenet/val/ILSVRC2012_img_val.tar -C imagenet/val\n",
+    "!rm -f imagenet/val/ILSVRC2012_img_val.tar\n",
+    "# We follow https://github.com/pytorch/examples/blob/main/imagenet/extract_ILSVRC.sh here\n",
+    "# get script from soumith and run; this script creates all class directories and moves images into corresponding directories\n",
+    "!cd imagenet/val/ && wget -qO- https://raw.githubusercontent.com/soumith/imagenetloader.torch/master/valprep.sh | bash\n",
```

Comment:
consider to take the code from here, looks more elegant.. https://github.com/sony/model_optimization/blob/main/tutorials/notebooks/keras/ptq/example_keras_imagenet.ipynb